src/types/events/base-events.ts

```diff
@@ -146,6 +146,7 @@ export interface AppMentionEvent {
   ts: string;
   channel: string;
   event_ts: string;
+  thread_ts: string;
```

Comment:
looks good to me but can you add the following fields as well?

* `subtype?: string` // bot_message etc
* `bot_id?: string`
* `username: string`

Also, it seems `user` is no longer available as far as I know. Can you update the field as optional (`user?: string`)?

Reply:
Done! Thanks for suggesting these changes.